packages/orderbook/test/js_api/gui.test.ts

```
@@ -1025,18 +1022,17 @@ ${dotrain}`;
 			assert.equal(deposits[1].address, '0x8f3cf7ad23cd3cadbd9735aff958023239c6a063');
 
-			let result = gui.getCurrentDeployment();
+			const result = gui.getCurrentDeployment();
 			const guiDeployment = extractWasmEncodedData<GuiDeploymentCfg>(result);
 			assert.equal(guiDeployment.deployment.order.inputs[0].vaultId, '0x29a');
 			assert.equal(guiDeployment.deployment.order.outputs[0].vaultId, '0x14d');
 		});
 
 		it('should throw error if given dotrain is different', async () => {
-			const gui = new DotrainOrderGui();
 			let testDotrain = `${guiConfig}
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'testDotrain' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)